addons/nuxeo-core-binarymanager-cloud/nuxeo-core-binarymanager-s3/src/main/java/org/nuxeo/ecm/core/storage/sql/S3BinaryManager.java

```diff
@@ -418,6 +413,8 @@ protected static boolean isMissingKey(AmazonClientException e) {
         return false;
     }
 
+    /** @deprecated since 11.1, now unused */
+    @Deprecated
     public static boolean isMD5(String digest) {
         return MD5_RE.matcher(digest).matches();
```

Comment:
Why don't you fallback on the method isValidDigest? It would allow to remove the private constant MD5_RE.